src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -46,7 +46,7 @@
     <PackageReference Include="Microsoft.ApplicationInsights.PerfCounterCollector" Version="2.20.0" />
     <PackageReference Include="Microsoft.ApplicationInsights.WindowsServer" Version="2.20.0" />
     <PackageReference Include="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" Version="2.20.0" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.34-11957" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.37" />
```

Comment:
Dev branch webjobs version for reference - https://github.com/Azure/azure-functions-host/blob/76591f722931f651caddeb0f6080d934545246dd/src/WebJobs.Script/WebJobs.Script.csproj#L51

Nuget.org - https://www.nuget.org/packages/Microsoft.Azure.WebJobs